plugins/catalog-import/api-report.md

```diff
@@ -299,6 +328,71 @@ export interface PreviewPullRequestComponentProps {
   title: string;
 }
 
+// @public (undocumented)
+export type RecursivePartial<T> = {
+  [P in keyof T]?: T[P] extends (infer U)[]
+    ? RecursivePartial<U>[]
+    : T[P] extends object
+    ? RecursivePartial<T[P]>
+    : T[P];
+};
+
+// @public
+export type ReviewResult =
+  | {
+      type: 'locations';
+      locations: Array<{
+        target: string;
+        entities: Entity[];
+      }>;
+      refreshed: Array<{
+        target: string;
+      }>;
+    }
+  | {
+      type: 'repository';
+      url: string;
+      integrationType: string;
+      pullRequest: {
+        url: string;
+      };
+      locations: Array<{
+        target: string;
+        entities: Entity[];
+      }>;
+    };
+
+// @public
+export type State =
```

Comment:
Veeeeery generic name x). There are quite a lot of questionable names here tbh, maybe worth extracting to a separate PR for bike shedding?

Reply:
aight, i'll try to pull that one out separately then

Reply:
Done, extracted all catalog-import concerns and may do a separate pr for that